apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -76,8 +76,7 @@ defmodule OMG.Eth.RootChain do
     contract = Config.maybe_fetch_addr!(contract, :payment_exit_game)
 
     static_array_return_types = [
-      {:array, {:tuple, [:bool, {:uint, 256}, {:bytes, 32}, :address, {:uint, 256}, {:uint, 256}]},
-       Enum.count(exit_ids)}
+      {:array, {:tuple, [:bool, {:uint, 256}, {:bytes, 32}, :address, {:uint, 256}, {:uint, 256}]}}
```

Comment:
:+1: and I don't know why I figured that, after stripping out the offest part from https://solidity.readthedocs.io/en/v0.5.13/abi-spec.html#use-of-dynamic-types, what remains should be decoded as a static array. It is still a dynamic array and encoded like you mentioned in #1235, the offset prefix is like "an other kind of dynamicness" involved. Not fully grasping this yet but makes sense.

However, since now this is fixed, the naming and comments I originally put there become bit confusing. Lemme suggest:
  - `static_array_return_types` -> `return_types`
  - `proceeds to decode the array as if it were static` -> `proceeds to decode the array without the offset`
  - `static_array_data::binary` -> `raw_array_data::binary`

Reply:
Updated in 2594ee448a66ff6f5873d83901ec424e2ba79bcb